### ISSUE TITLE
iOS build fail : path '.../myApp.ipa' not found

### ISSUE THREAD

**Issue by mat3000** [dpa99c/cordova-plugin-firebasex#119]
i start a new project with ionic 3 :

`ionic start helloWorld blank --type=ionic-angular`

and prepare ios

`ionic cordova prepare ios`
 
i install firebasex like this :

`ionic cordova plugin add cordova-plugin-firebasex`

but after running this command :

`ionic cordova run ios -l -c --verbose`

i got this error :

```
[cordova] Prepared iOS project successfully
[cordova] Executing script found in plugin cordova-plugin-firebasex for hook "after_prepare": plugins/cordova-plugin-firebasex/scripts/after_prepare.js
[cordova] Preparing Firebase on iOS
> native-run ios --app platforms/ios/build/device/MyApp.ipa --device --forward 8100:8100 --forward 35729:35729 --forward 53703:53703 --verbose
[native-run] 2019-08-28T09:47:34.488Z native-run Caught fatal error: { Error: Path 'platforms/ios/build/device/MyApp.ipa' not found
[native-run]     at Object.run (/Users/mat3000/.nvm/versions/node/v10.16.1/lib/node_modules/native-run/dist/ios/run.js:23:15)
[native-run]     at Object.run (/Users/mat3000/.nvm/versions/node/v10.16.1/lib/node_modules/native-run/dist/ios/index.js:14:15)
[native-run]     at process._tickCallback (internal/process/next_tick.js:68:7)
[native-run]     at Function.Module.runMain (internal/modules/cjs/loader.js:834:11)
[native-run]     at startup (internal/bootstrap/node.js:283:19)
[native-run]     at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3) code: undefined, exitCode: 1, data: undefined }
[native-run] ERR_UNKNOWN: Path 'platforms/ios/build/device/MyApp.ipa' not found
[ERROR] An error occurred while running subprocess native-run.
        
        native-run ios --app platforms/ios/build/device/MyApp.ipa --device --forward 810... exited with exit code 1.
```
if i remove cordova-plugin-firebasex plugin, build work fine

my config :
```
Ionic:

   Ionic CLI          : 5.2.6 (/Users/mat3000/.nvm/versions/node/v10.16.1/lib/node_modules/ionic)
   Ionic Framework    : ionic-angular 3.9.5
   @ionic/app-scripts : 3.2.2

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : android 8.0.0, ios 5.0.1
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 4.1.1, (and 7 other plugins)

Utility:

   cordova-res : 0.6.0 
   native-run  : 0.2.8 

System:

   Android SDK Tools : 26.1.1 (/Users/mat3000/Library/Android/sdk/)
   ios-deploy        : 1.9.4
   ios-sim           : 8.0.2
   NodeJS            : v10.16.1 (/Users/mat3000/.nvm/versions/node/v10.16.1/bin/node)
   npm               : 6.10.3
   OS                : macOS Mojave
   Xcode             : Xcode 10.3 Build version 10G8
```

thanks for your help

**Comment by mat3000** [comment]
I resolve issue with downgrade version of cordova-plugin-firebasex to v6.0.6

**Comment by rolinger** [comment]
@dpa99c I am getting the same issue with my new project build.  The `ionic cordova build iOS` gets to the same `Preparing Firebase on iOS` and then cleanly exits the build process - it never completes the build thus `platforms/ios/build` directory and files are never created.  Been chasing this issue for 24 hours now.  

My app is an older Ionic-v1 app and I just upgraded the Ionic cli from v1.7.14 to v5.2.6.  On Android, even using `Cordova-plugin-ionic-webview` (which deploys natively when creating a new project with v5.2.6), my app builds and runs on an Android device just fine.  Everything works normally.

But not on iOS.  As stated above, it never goes through the full build process.  HOWEVER, I can run the app, with FirebaseX, directly from Xcode deployed to my test iPhone.  Even with `cordova-plugin-ionic-webview` Xcode can deploy the app to the test iPhone - though many things don't work.  So maybe its a Firebasex cordova build issue versus just an error/issue with the plugin.

For now though, I have removed the `Cordova-plugin-ionic-webview` and added an older trusty `Webview-plus` - and when deployed from Xcode it all works as normal - but still can't get a proper cli build to work.  I don't believe the `Webview` and `build` issues are related but I thought I would mention it anyway.

`Cordova-plugin-firebasex @6.0.7` - I know 6.0.6 worked previously and I see @mat3000 went back to 6.0.6 and got it work, but I just fixed all my code for the breaking changes.  Ugh!

```
Ionic:

   Ionic CLI         : 5.2.6 (/usr/local/lib/node_modules/ionic)
   Ionic Framework   : ionic1 1.0.0
   @ionic/v1-toolkit : 1.0.22

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : ios 5.0.1
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.1.3, (and 32 other plugins)

Utility:

   cordova-res : 0.6.0 
   native-run  : 0.2.8 

System:

   ios-deploy : 1.9.4
   ios-sim    : 8.0.2
   NodeJS     : v10.16.3 (/usr/local/bin/node)
   npm        : 6.11.2
   OS         : macOS High Sierra
   Xcode      : Xcode 10.1 Build version 10B61
```

**Comment by rolinger** [comment]
@dpa99c - yup, I downgraded to 6.0.6 on my iOS platform and everything ALMOST worked.  It now hangs at `/SharedPrecompileHeaders/122334566421114945895838/MyApp-Prefix.pch.dia` during the build - as reported here: https://github.com/dpa99c/cordova-plugin-firebasex/issues/5#issuecomment-504001501 - something is wrong in 6.0.7 for build for iOS only when doing `ionic cordova build iOS`, 6.0.7 worked just fine for Android.

**Comment by dpa99c** [comment]
The cause is the same as #101 and is fixed on master by PR #121 which will be released shortly in v6.0.8
